demos/espressif/esp32_devkitc_esp_wrover_kit/make/sdkconfig

```diff
@@ -116,8 +121,227 @@ CONFIG_ESP32_APPTRACE_ENABLE=
 CONFIG_ESP32_APPTRACE_LOCK_ENABLE=y
 
 #
-# FreeRTOS SystemView Tracing
+# Bluetooth
 #
+CONFIG_BT_ENABLED=y
```

Comment:
Do we want bluetooth enabled in master?

Reply:
We do need it for GA, I could remove it but I think it is good to keep both branches in Sync.